pkg/reconciler/space/resources/build_service_account.go

```diff
@@ -24,22 +24,54 @@ import (
 // BuildServiceAccountName gets the name of the service account for given the
 // space.
 func BuildServiceAccountName(space *v1alpha1.Space) string {
-	return "kf-build-service-account"
+	return space.Spec.Security.BuildServiceAccount
```

Comment:
I think we should always create/sync the default build service account, that way the user can override if necessary to opt-out of syncing.

Reply:
When you say default, do you mean `DefaultBuildServiceAccountName` or do you mean `default`?